.env

```diff
@@ -34,10 +34,12 @@ TOPOLOGY_APP_SERVICE=topology
 NGINX_HTTP2=http2
 NGINX_CLIENT_BODY_SIZE=5M
 NGINX_IP6=True
-NGINX_GZIP=off
+NGINX_ADMIN_ALLOW_NETWORK=all
+NGINX_SSL_CONFIG=gzip on;gzip_comp_level 6;gzip_proxied any;gzip_min_length 1000;gzip_types text/plain image/svg+xml application/json application/javascript text/xml text/css application/xml application/x-font-ttf font/opentype;
```

Comment:
how does this work?
is it going to be expanded? Or it doesn't need to be on different new lines?
Wouldn't something like this be hard to manage?

Reply:
Maybe you can create an issue to improve this later as well

Reply:
It doesn't need to be expanded. I can move them to seperate variables if needed.
Please note that even right now, in the ansible-openwisp2 I see the implementation of gzip is in the same line.
([screenshot here](https://user-images.githubusercontent.com/24985760/58369162-411cf100-7f14-11e9-9d65-6f2b59a1091c.png).)

Reply:
the resulting file is not very relevant, what is relevant is the format we use to keep the variables, we need something a bit more flexible, having every value on one var would be painful.

I suggest creating an issue for this specific thing, we can investigate later how to improve it